wheel/python/clvm_rs/at.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+from typing import Optional, cast
+
+from .clvm_storage import CLVMStorage
+
+
+def at(obj: CLVMStorage, position: str) -> Optional[CLVMStorage]:
```

Comment:
there should be unit tests for this function.
Also, I don't see any circumstance where this function returns `None`. Should it not just return `CLVMStorage`?

Reply:
The `None` is below, line 31.

Reply:
Tests are in `wheel/python/tests/test_program.py` https://github.com/Chia-Network/clvm_rs/blob/0ddb2668434de8a19222eed56fef060f9e7ca44b/wheel/python/tests/test_program.py#L14

For some reason, github decided there were too many changes in this (new) file and elides it from the default view of this patch.